front/app/components/admin/PostManager/components/FilterSidebar/FilterRadioButton/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import { Box } from '@citizenlab/cl2-component-library';
+import { hideVisually } from 'polished';
+import styled from 'styled-components';
+
+const HiddenRadio = styled.input.attrs({ type: 'radio' })`
+  ${hideVisually()};
+`;
+
+const Label = styled.label<{ selected: boolean }>`
```

Comment:
https://www.notion.so/govocal/Add-BoxProps-to-Label-1629663b7b2680dca934c5e48504daca?pvs=4